typing/typedecl.ml

```diff
@@ -1644,6 +1644,8 @@ let transl_value_decl env loc valdecl =
   | [] ->
       raise (Error(valdecl.pval_loc, Val_in_structure))
   | _ ->
+      let msg = Printf.sprintf "the signature for extern %s cannot be checked" valdecl.pval_name.txt in
```

Comment:
for "external"?

Should this be restricted to type-checking implementation, or should this also fire when type-checking signature? (No strong opinion.)